### PR TITLE
Issue 63 make all new tables and DAOs

### LINKED ISSUE #62: Create Room DAO for Clubs info table
Create a DAO to enable CRUD operation on the Clubs table in the Room Database

DoD: DAO created and unit test added for CRUD operation

### LINKED ISSUE #63: Create DAO for users
DOD: have a users table and DAO

### LINKED ISSUE #64: Create DAO for sustainability table
DOD: a table  and DAO for sustainability

### LINKED ISSUE #65: Subscriptions DAO
DOD: Subscriptions DAO, table complete

### PR DESCRIPTION
**Pull request by isaakwheeler**
made all of the tables and daos

### CODE REVIEW

**Review by haythamodeh — APPROVED** [review]
approved

File: app/src/main/java/edu/cascadia/mobas/campusguidebook/data/model/ClubModel.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    public void setClubName(String ClubName) {
+        this.ClubName = ClubName;
+    }
+
+
+    public int getId() {
+        return id;
+    }
+    public void setId(int ID) {
+        this.id = ID;
```
Comment (gomezfox) [inline app/src/main/java/edu/cascadia/mobas/campusguidebook/data/model/ClubModel.java:41]: Autoincrementing primary keys cannot be modified afaik

**Review by odjuliarso — APPROVED** [review]

File: app/src/main/java/edu/cascadia/mobas/campusguidebook/data/model/ClubModel.java (hunk 1 above)
Comment (isaakwheeler) [reply app/src/main/java/edu/cascadia/mobas/campusguidebook/data/model/ClubModel.java:41]: that is true but the program gives an error if you only have a get method

**Review by gomezfox — APPROVED** [review]
Very clean looking. Good work!

**Pull request merged**